internal/server/server.go

```diff
@@ -196,3 +196,19 @@ func newInternalServer() (*http.Server, error) {
 
 	return &server, nil
 }
+
+// traceMethod - determines if opentracing should be enabled for the grpc method
+func traceMethod(ctx context.Context, fullMethodName string) bool {
+	donotTraceMethods := []string{ // Should be a very small list
```

Comment:
nit: I understand the list is small for now, but still better to keep this as a map here for a constant time lookup.

Reply:
that being said, can this being a `var map[string]bool` init'ed on boot instead of a separate function?